source/standards/logging.html.md.erb

```diff
@@ -1,31 +1,104 @@
 ---
 title: Logging
-expires: 2017-09-01
+expires: 2017-12-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-## Shared logging platform
+## Short-term querying of logs
 
-The service support and operations team is investigating whether
-we should run a shared logging platform for product teams at GDS.
+For the purposes of incident management and debugging, it's useful to
+have a system for storing and querying the most recent logs produced
+by your application and infrastructure.
 
-If you're about to start some work on logging, ask in the
-[#tech-ops-forum Slack channel](https://govuk.slack.com/messages/tech-ops-forum/)
-whether there's something you can use.
+In general, tools for querying logs are not a good fit for long-term
+archival, and vice versa.
 
-## Sumo Logic
+### Shared Elasticsearch, Logstash and Kibana (ELK) stack
 
-Some teams have used [Sumo Logic](https://www.sumologic.com/)
-successfully for storing and querying logs.
+GDS has a shared account with logit, which offers on-demand ELK stacks
```

Comment:
Is it worth putting a link to Logit here? I remember Googling for them, and realising that their value wasn't `p` - as such, it was difficult to work out what the log-odds were of me finding their site in the first place.